queries/nu/highlights.scm

```diff
@@ -0,0 +1,358 @@
+; ---
+; keywords
+[
+  "let"
+  "let-env"
+  "mut"
+  "const"
+  "hide-env"
+  "source"
+  "source-env"
+  "overlay"
+  "register"
+] @keyword
+
+[
+  "if"
+  "else"
+  "match"
+] @keyword.conditional
+
+[
+  "loop"
+  "while"
+  "break"
+  "continue"
+] @keyword.repeat
+
+[
+  "def"
+  "do"
+] @keyword.function
+
+"return" @keyword.return
+
+[
+  "try"
+  "catch"
+  "error"
+] @keyword.exception
+
+"module" @keyword.import
+
+[
+  "alias"
+  "export-env"
+  "export"
+  "extern"
+] @keyword.modifier
+
+(hide_mod
+  "hide" @keyword)
+
+(decl_use
+  "use" @keyword.import)
+
+(ctrl_for
+  "for" @keyword
+  "in" @keyword)
+
+(overlay_list
+  "list" @keyword.import)
+
+(overlay_hide
+  "hide" @keyword.import)
+
+(overlay_new
+  "new" @keyword.import)
+
+(overlay_use
+  "use" @keyword.import
+  "as" @keyword)
+
+(ctrl_error
+  "make" @keyword.import)
+
+; ---
+; literals
+(val_number) @number
+
+(val_duration
+  unit: _ @variable.parameter)
+
+(val_filesize
+  unit: _ @variable.parameter)
+
+(val_binary
+  [
+    "0b"
+    "0o"
+    "0x"
+  ] @number
+  digit: [
+    "," @punctuation.delimiter
+    (hex_digit) @number
+  ]) @number
+
+(val_bool) @constant.builtin
+
+(val_nothing) @constant.builtin
+
+(val_string) @string
+
+arg_str: (val_string) @variable.parameter
+
+file_path: (val_string) @variable.parameter
+
+(val_date) @number
+
+(inter_escape_sequence) @string.escape
+
+(escape_sequence) @string.escape
+
+(val_interpolated
+  [
+    "$\""
+    "$\'"
```

Comment:
Please remove the unnecessary backslash here, for the single quote. Also needs a string highlight for just `"'"` (a lone single quote)